docs/howto/local-checkouts.md

```diff
@@ -0,0 +1,343 @@
+---
+title: Work with lakeFS data locally
+description: How to work with lakeFS data locally
+parent: How-To
+---
+
+# Working with lakeFS Data Locally
+
+lakeFS is a scalable data version control system designed to scale to billions of objects, and as such it has no notion of
```

Comment:
Nitpick: I'd lead with the positive (i.e. instead of "lakeFS doesn't do X, however sometime it's needed", go with "with lakeFS you can do Y in addition to X!")

Reply:
Maybe add another short paragraph to act as TL;DR?

> This document explains `lakectl local`, a command that lets you clone data from lakeFS to a local directory, keep it in sync, ...